install.sh

```diff
@@ -135,17 +135,15 @@ install_packages( )
   # dependencies needed for rocblas and clients to build
```

Comment:
Comment is misleading: Does this part of the script add dependencies for `rocblas` _and_ `clients` to build? If so, why is `libgomp` being removed? Or is this part of the script only for `rocblas`, and not for `clients`?

Reply:
This is the part of the script for building without the client. I have fixed this in my latest commit